feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/domain/AccountInteractorImpl.kt

```diff
@@ -190,4 +191,14 @@ class AccountInteractorImpl(
             }
         }
     }
+
+    override suspend fun switchToNotDeactivatedAccountIfNeeded() {
+        val metaAccount = accountRepository.getSelectedMetaAccount()
+        if (metaAccount.status == LightMetaAccount.Status.DEACTIVATED) {
+            val metaAccounts = getMetaAccounts()
+            if (metaAccounts.isNotEmpty()) {
```

Comment:
I suggest to avoid nested ifs by doing easly return:

``` kotlin
val metaAccount = accountRepository.getSelectedMetaAccount()
if (metaAccount.status != LightMetaAccount.Status.DEACTIVATED) return
val metaAccounts = getMetaAccounts()
if (metaAccounts.isEmpty()) return
// ...
```